code/g.py

```diff
@@ -269,6 +271,35 @@ def new_game(difficulty_name):
     # Reset music
     mixer.play_music("music")
 
+internal_id_version = None
+
+def to_internal_id(obj_type, obj_id):
+    if internal_id_version:
```

Comment:
As I understand this, then this code can only work for convert ids from a legacy (pickle) savegame to a new style savegame (as that is the only time we call `to_internal_id` on load).